cmd/odfcli/set/set.go

```diff
@@ -0,0 +1,17 @@
+package set
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// SetCmd represents the set command
+var SetCmd = &cobra.Command{
+	Use:                "set",
+	Short:              "Set ODF configuration",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
```

Comment:
I think the `Args` field doesn't matter here. From what I understand of Cobra, subcommands don't count as arguments. For the multus CLI in Rook, which is comparable, I left this field blank

https://github.com/rook/rook/blob/0f6e7ee921e98776c31b4b0912e69f093caeccfc/cmd/rook/userfacing/multus/multus.go#L30-L41

Reply:
right, cobra don't count subcommands as args